README.md

```diff
@@ -99,6 +109,8 @@ jobs:
           label-name:
           # 최소 리뷰어 수를 지정할 수 있습니다. 기본은 1명입니다.
           reviewers-number:
+          # @todo in korean.
```

Comment:
@plm9606 translate this in Korean please
`The merge method, one between "merge", "squash" or "rebase". Default is "merge".`